src/includes/enriching-events/set-user/dart.mdx

```diff
@@ -5,3 +5,9 @@ Sentry.configureScope(
   (scope) => scope.user = SentryUser(id: '1234', email: 'jane.doe@example.com'),
 );
 ```
+
+<Note>
+
+If you do not provide a user identity, the SDK falls back to `installationId`, which the SDK once randomly generates during an app's installation.
+
+</Note>
```

Comment:
we should create a new file (copy from this one) called `flutter.mdx` and add this information only for Flutter, the Dart SDK does not have such fallback since there's no offline caching feature.

Reply:
Good catch, thanks.